src/bindings/jna/gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
```

Comment:
Please avoid unrelated changes in PRs.

Reply:
I've created #4037 and will remove this change from this PR